conf/messages.cy

```diff
@@ -43,18 +52,22 @@ global.error.404.line1=Os gwnaethoch deipio’r cyfeiriad gwe, gwiriwch ei fod y
 global.error.404.line2=Os gwnaethoch ludo’r cyfeiriad gwe, gwiriwch eich bod wedi copïo’r cyfeiriad yn llawn.
 global.error.404.line3=<a class="govuk-link" href="{0}">Anfon eich dogfennau i’w gwirio gan y tollau</a>
 
-global.error.500.title=Sorry, there is a problem with the service
-global.error.500.heading=Sorry, there is a problem with the service
+global.error.500.title=Mae’n ddrwg gennym, mae problem gyda’r gwasanaeth
+global.error.500.heading=Mae’n ddrwg gennym – mae problem gyda’r gwasanaeth
 global.error.500.line1=Try again. If the issue persists and you need help, you can <a class="govuk-link" target="_blank" rel="noopener noreferrer" href="{0}">report the problem (opens in a new window or tab)</a>. We will respond to you within two working days. 
```

Comment:
The rest of the translations can be found on the story - the newest spreadsheet (the one added yesterday)

Reply:
Can also get the Hold SLA content from the same spreadsheet - still needs added in